### PR TITLE
Update ipamod.json

### PR DESCRIPTION
**Pull request by huutanphp**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by huutanphp** [comment]
Register ipamod.is-a.dev with A record pointing to 3.106.238.127

File: domains/iosviet.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "huutanphp",
+           "email": "",
+           "discord": "595368972594184235",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.UUB3u8JnQhweiOtS8ICf09sYk-WZL0haW_S50x1JgijmaiIBNCcX8hzE5xJVR2QXtiR_DfbseMXMtHVq5p_9SHf-qyiD8LKd22TKgorKH30PxP2Ej6Qw9LhbQKdI7EuAlBBGKyKm9gLWd8HYVeQlXMuya5X6AEKQS4WoH_GuX3itLnFUKf1hFZpEb5lJrdRKrvbTQgTXkLHsXggUi4Crcdfav4lmsA7DidmmfqpwXVy90TXdluqExc70JrtIY_9q0NDXwBe5boFOFOEYh7GW9MG0QHhor2Xibnd0pi5P5jAt9fUCGhUMMHKWTLBMjLumgswe8XEW4JXQGwmBBVMt_A.KpU_DlcxDclPoZwMO0H7rA.igZyUYSqQEyzpCfjGYAQivF9Jk8A6TzvA6sf2Ju7c4jP617xNtd7ZtTz2Wy9eYeSyF0ZDeEXn5RSWphywrSaIB6jhvzuEQ4xERnRW8ozdN0.xLh6AUpP7bDHQ7P8DwaZNw"
+        },
+    
+        "record": {
+            "A": ["3.106.238.127"]
```
Comment (DEV-DIBSTER) [inline domains/iosviet.json:10]: Just get a nginx landing page, can you send a preview of the site?

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/iosviet.json (hunk 1 above)
Comment (huutanphp) [reply domains/iosviet.json:10]: Screen Shot 2024-09-22 at 04 53 55

Comment (DEV-DIBSTER) [reply domains/iosviet.json:10]: Yeah that's just the default page, there's no content. Is there a site or not?

Comment (huutanphp) [reply domains/iosviet.json:10]: E3836C63-0E82-4097-9603-207E42C7BE6B

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**